Modules/EffortTracking/Resources/views/show.blade.php

```diff
@@ -59,11 +59,13 @@ class="text-theme-body text-decoration-none mb-2 mb-xl-4 align-items-center">
                         <i class="fa fa-spinner fa-spin ml-2 d-none"></i>
                         <i class="ml-2 font-weight-bold fa fa-refresh c-pointer" aria-hidden="true"
                             data-url="{{ route('effort-tracking.refresh', $project) }}"></i>
-                        @if ($project->last_updated_at)
-                            <div class="fz-14 float-right mr-3 mt-1">
-                                {{ config('project.meta_keys.last_updated_at.value') . __(': ') . (Carbon\Carbon::parse($project->last_updated_at)->format('D g:i a, dS M Y'))}}
+                            <div class="fz-14 float-right">&nbsp;&nbsp;&nbsp;
+                            <strong>Last refreshed at:</strong>{{ (Carbon\Carbon::parse($project -> last_updated_at) -> setTimezone('Asia/Kolkata') -> format(' Y-M-d , D h:i:s A')) }}
+                            </div > 
+                            <div class="fz-14 float-right">
+                            <strong>Timeline:</strong>{{ (Carbon\Carbon::parse($project -> client -> month_start_date) -> format('dS M')) }}                       
+                            -{{ (Carbon\Carbon::parse($project -> client -> month_end_date) -> format('dS M')) }}              
```

Comment:
Don't use space when using arrow 
```suggestion
                            -{{ (Carbon\Carbon::parse($project->client->month_end_date)->format('dS M')) }}              
```